test/types/index.ts

```diff
@@ -15,6 +15,11 @@ const span = agent.currentSpan
 if (span) span.end()
 const traceparent = agent.currentTraceparent
 if (traceparent) traceparent.split('-')
+const currentTraceIds = agent.currentTraceIds
+let traceId = currentTraceIds['trace.id'] || ''
+traceId += '-' + currentTraceIds['transaction.id'] === undefined
+  ? currentTraceIds['transaction.id']
+  : currentTraceIds['span.id']
```

Comment:
I didn't catch this issue before, but as far as I see this statement will always be `false` because there's implicit parentheses around `('-' + currentTraceIds['transaction.id'])` 🤔 I might read the code wrong, but I think what you meant was something like this right?

```suggestion
traceId += '-' + (currentTraceIds['transaction.id'] === undefined
  ? currentTraceIds['transaction.id']
  : currentTraceIds['span.id'])
```

Though I'm not sure it matters much since this is the TypeScript tests, so we don't really care about the result 😅

Reply:
Good catch 👍

though as long as it compiles we should be fine 😄